Makefile

```diff
@@ -1,7 +1,7 @@
 SHELL := /bin/bash
 export GO111MODULE ?= on
-export VERSION ?= v1.1.0
-export ECE_VERSION ?= 2.8.0-ms48
+export VERSION ?= v1.2.0
+export ECE_VERSION ?= 2.8.0
```

Comment:
When is 2.8.0 being released? Should this not be `2.8.0-ms49` for now?

Reply:
According to our schedule, on the 26th. The spec shouldn't change since we've already gone over the freeze and it'll only be bug fixes, nothing affecting the spec.